railties/lib/rails/test_unit/testing.rake

```diff
@@ -1,12 +1,12 @@
-require "rails/test_unit/runner"
-
 task default: :test
 
 desc "Runs all tests in test folder"
 task :test do
   $: << "test"
-  args = ARGV[0] == "test" ? ARGV[1..-1] : []
-  Rails::TestRunner.run(args)
+
+  ARGV.shift if ARGV[0] == "test"
+
+  Rails::TestRunner.run
```

Comment:
why the runner dont receive the args?
I remember I initially had the `ARGV.shift` too, but I think, it didnt work when you do: `bundle exec rake test db:migrate` , dont fully remember why.